README.md

```diff
@@ -1,2 +1,71 @@
-# figshare
-github actions for figshare
+# Figshare
+
+**GitHub Actions for Figshare.**
+
+## Publish an article on Figshare
+To publish an article at [Figshare](https://figshare.com/). See the below example.
+
+```hcl
+workflow "Create DOI" {
+  resolves = "publish"
+}
+
+action "create" {
+  uses = "./create"
```

Comment:
this would have to change so it reads `popperized/figshare/create@master`. And similarly on the other references to actions contained in this README